lab_5_scrapper/scrapper.py

```diff
@@ -140,10 +287,47 @@ def get_search_urls(self) -> list:
         Returns:
             list: seed_urls param
         """
+        return self.config.get_seed_urls()
 
 
 # 10
 # 4, 6, 8, 10
+class CrawlerRecursive(Crawler):
```

Comment:
your crawler is not recursive. recursive crawler accepts your website https://www.sbras.info/news and does all the things - collects ALL links from the page, with the goal of finding article URLs. Then checks EACH of these ALL links whether they have article URLs inside. Then checks EVERY of these EACH links, etc. 

- It stops when it has enough URLs for articles
- It can restart from the moment is stopped last time. Remember, articles are created on filesystem only after HTML parser visits pages, and this happens only after crawler finishes.
- It knows what was visited and what was not (has some persistent state)

Reply:
I think, I've implemented it!